frontend/src/components/delete/Delete.js

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import axios from "axios";
+import "./Delete.css";
+
+function Delete() {
+
+    const email = sessionStorage.getItem("email");
+    const history = useHistory();
+    const [password, setPassword] = useState("");
+    const [verifyPassword, setVerifyPassword] = useState("");
+    
+    const handleChange = (event) => {
+        if (event.target.id === "password"){setPassword(event.target.value)};
+        if (event.target.id === "verifyPassword"){setVerifyPassword(event.target.value)};
+    }
+    const sendDataToExpress = (event) => {
+        event.preventDefault();
+        console.log(password);
+        let loggedInPassword = sessionStorage.getItem('password');
+        console.log(loggedInPassword);
+        if ((password == loggedInPassword) && (password == verifyPassword)){
+            axios.delete('api/users/', {data: {
+                id: sessionStorage.getItem("userId"),
+            }}).then((res) => {
+                console.log(res.data);
+                if (res.data.message === "Account deleted") {
```

Comment:
Remember to check res.data.status for "OK" and simply print res.data.message in the alert :)